tests/lib/utils.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (C) 2018  Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+#
+# Red Hat Author(s): Vendula Poncova <vponcova@redhat.com>
+#
+from textwrap import dedent
+from mock import Mock
+from pyanaconda.dbus.constants import DBUS_MODULE_NAMESPACE
+
+
+def check_kickstart_interface(test, interface, ks_in, ks_out):
+    """Test the parsing and generating of a kickstart module.
+
+    :param test: instance of TestCase
+    :param interface: instance of KickstartModuleInterface
+    :param ks_in: string with the input kickstart
+    :param ks_out: string with the output kickstart
+    """
+    callback = Mock()
+    interface.PropertiesChanged.connect(callback)
+
+    # Read a kickstart,
+    if ks_in is not None:
+        ks_in = dedent(ks_in).strip()
+        result = interface.ReadKickstart(ks_in)
+        test.assertEqual({k: v.unpack() for k, v in result.items()}, {"success": True})
+
+    # Generate a kickstart
+    ks_out = dedent(ks_out).strip()
+    test.assertEqual(ks_out, interface.GenerateKickstart().strip())
+
+    # Test the properties changed callback.
+    if ks_in is not None:
+        callback.assert_any_call(DBUS_MODULE_NAMESPACE, {'Kickstarted': True}, [])
+    else:
+        test.assertEqual(interface.Kickstarted, False)
+        callback.assert_not_called()
```

Comment:
Please move this file to `pyanaconda_tests/__init__.py` file instead.
This lib directory is for general use stuff not for just unit tests.

Reply:
Fixed.